centreon/tests/e2e/features/Dashboards/17-clock-timer-widget-configuration/index.ts

```
@@ -164,7 +164,12 @@ Then(
           timeZone: 'Europe/Paris'
         });
 
+<<<<<<< HEAD
         cy.log(`Current hour (hoursNow): ${hoursNow}`);
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)